.env.example

```
@@ -13,6 +13,13 @@ NEXT_PUBLIC_APP_URL=https://babylon.market
 NEXT_PUBLIC_BLOG_URL=https://blog.babylon.market
 # Toggle global waitlist/coming-soon mode (true restricts app to landing)
 WAITLIST_MODE=false
+
+# ============================================================================
+# Admin Configuration
+# ============================================================================
+# Email domain for automatic admin access (users with this email domain get admin)
+# Example: elizalabs.ai will grant admin to all users@elizalabs.ai emails
+ADMIN_EMAIL_DOMAIN=elizalabs.ai  # Optional: Auto-promote users with this email domain to admin
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)